src/popup/searchModule.js

```diff
@@ -105,6 +105,7 @@ export function checkValidationError(apiResponse) {
     showNotification('Not a valid search query', 'negative', 'snackbar-bookmarks');
     removeLoaderAnimation();
     restoreInitialContent('primary');
+    elements.loadMoreButtonWrapper.classList.add('removeLoadMore');
```

Comment:
I am sure that you have missed something because in the current PR what you are essentially doing is to only remove the load more button in case of validation error. But that's not the only thing we want.

As described in the issue, the _Search section should reset after an invalid search query_. The GIF you posted is showing the correct results but this PR does not replicate it.

Also, please uncheck the 5th checkbox of the PR description.